core/amber/src/main/scala/edu/uci/ics/texera/workflow/WorkflowCompiler.scala

```diff
@@ -176,21 +176,14 @@ class WorkflowCompiler(
       logicalPlan
     )
     // - resolve the file name in each scan source operator
-    logicalPlan.resolveScanSourceOpFileName(Some(errorList))
+    logicalPlan.resolveScanSourceOpFileName(None)
```

Comment:
why not collecting errors here?

Reply:
When the None is given, these functions are going to directly throw errors out. The outer handles the exception using errorHandler.